pkg/features/kube_features.go

```diff
@@ -896,7 +896,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CSIMigrationRBD: {Default: false, PreRelease: featuregate.Alpha}, // Off by default (requires RBD CSI driver)
 
-	CSIMigrationvSphere: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.28
+	CSIMigrationvSphere: {Default: true, PreRelease: featuregate.GA}, // LockToDefault when supported CSI driver is available
```

Comment:
Can you clarify what is a supported CSI driver, i.e., raw block, windows, xfs should be GA in CSI driver

Reply:
done